js/widgets/fixedToolbar.js

```diff
@@ -185,39 +170,42 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 		},
 
 		show: function( notransition ) {
-			var hideClass = "ui-fixed-hidden",
+			var hideClass = "ui-toolbar-fixed-hidden",
```

Comment:
I don't think there is any point in saving this to a var seems like its only used twice and its just a single class name?